src/native/eventpipe/ep-stream.h

```diff
@@ -355,5 +355,26 @@ ep_ipc_stream_writer_write (
 	uint32_t bytes_to_write,
 	uint32_t *bytes_written);
 
+/*
+ * IpcContinuationStream
```

Comment:
Defining a new struct to wrap IpcStream seems unnecessary. Couldn't we just make the continuation_stream field in the session be type IpcStream*?

Reply:
I think IpcStream is meant to be a base class for different types of stream implementations, for example socket stream vs named pipe stream vs websocket stream. Its a bit less obvious because even though derived types have different implementations they are all reusing the same 'DiagnosticsIpcStream' type name. At some point in the future we could rename them SocketIpcStream, NamedPipeIpcStream, etc to make the inheritance pattern clearer. Sidestepping the naming, unless we are implementing a new transport I wouldn't expect a new stream derived type is needed.